src/test/java/io/getunleash/integration/TestCaseVariant.java

```diff
@@ -22,7 +22,14 @@ public String getToggleName() {
 
     public Variant getExpectedResult() {
         if (expectedResult.getName().equals("disabled")) {
-            return Variant.DISABLED_VARIANT;
+            Variant clone =
+                    new Variant(
```

Comment:
This seems really dumb, is there a better way to clone an object?

Reply:
You can have a clone method that does this for you, but the code will be there... I remember there were some utility methods, but atm I think this is fine